truffle-config.js

```diff
@@ -43,11 +43,12 @@ module.exports = {
     // tab if you use this network and you must also set the `host`, `port` and `network_id`
     // options below to some value.
     //
-    // development: {
-    //  host: "127.0.0.1",     // Localhost (default: none)
-    //  port: 8545,            // Standard Ethereum port (default: none)
-    //  network_id: "*",       // Any network (default: none)
-    // },
+    development: {
+     host: "127.0.0.1",     // Localhost (default: none)
+     port: 8545,            // Standard Ethereum port (default: none)
+     network_id: "*",       // Any network (default: none)
+     blockTime: 1,          // Default is 0 (a block is created when a transaction is received)
```

Comment:
Can we use a different port than 8545, which we use for the IOV-Core ganache that has a different configuration? https://github.com/iov-one/iov-core/blob/master/scripts/ethereum/start.sh
This is running almost all the time on my machine.

Reply:
Isn't ganache-cli started from scripts/test.sh without those settings?

Reply:
It is, but truffle uses them to know which network to connect. I'll update.

Reply:
But than `blockTime: 1` is the wrong value as ganache-cli defaults to 0 (_a block is created when a transaction is received_)

Reply:
Yep, the truffle client doesn't need the block time.

Reply:
okay, so it should either be unset or set to 0, right?